### ISSUE TITLE
Jetform/Booking Bug  Check In/Check Out 

### ISSUE THREAD

**Issue by nreljed** [CrocoBlock/suggestions#7816]
Hi,

```
Note : This bug has nothing to do with jetbooking version 3.6.0 /3.6.0.1
it exists before and after these versions I am on version 3.5.2
```

I've encountered a bug in the Check In/Check Out field of JetBooking/JetFormBuilder. I'm not sure if this is an issue with JetBooking, JetFormBuilder, or both, but here's the problem:

-Create a simple booking form with JetForm.
-Make sure to book days before and days after and leave for example an unbooked day between them to reproduce the famous bug
-Select the isolated day in the Check In/Check Out field 

![jetnooking-jetformbuilder bug](https://github.com/user-attachments/assets/716af5e0-b1f6-4f39-b057-896371865dad)

You'll notice that the calendar freezes on both desktop and mobile. There's no way to exit this loop; it's as if this scenario wasn't anticipated. 
The solution should be to allow clicking on the problematic day again to undo and deselect it, which isn't currently possible.

I suggest implementing a more modern Check In/Check Out functionality with an option to clear dates, similar to Airbnb. This would be more user-friendly and logical. This modification shouldn't require extensive development time. Please review the attached gif 

![airbnb](https://github.com/user-attachments/assets/b1e80994-b167-4e22-80fa-80d88dc690a7)

Thank you for considering this enhancement.

if you have a mini code in jetformbuilder to insert a clear dates button if Check In/Check Out is not empty I take it , 
while waiting for an appropriate answer to this scenario

**Comment by nreljed** [comment]
just to be sure that the problem exists and to try to understand how things work,

after analyzing which component jetform / booking use and I find that it is jquery-date-range-picker

[so going directly to the component page or a clone ](https://longbill.github.io/jquery-date-range-picker/)I actually see that it does not present the blocking anomalies on a date encountered

and indeed there are reset clear date mechanics and a whole bunch of possibilities

![test date picker ](https://github.com/user-attachments/assets/f130ace7-56bc-4037-ae43-a0e2df081ff8)


it would not be particularly hard or long to update this component or better replicate that of airbnb

as to why no one has ever noticed this abnormal behavior I have no answer, it is possible that the date-picker has been modified to disable certain things or simply integrated with the basic options

**Comment by Crocoblock** [comment]
Added the `clear button` option https://tppr.me/feBlvf, https://tppr.me/9cfwoy, https://tppr.me/2v0RxL
was added and released in JetBooking 3.6.3

**Comment by ofmarconi** [comment]
@nreljed pls check this: https://github.com/Crocoblock/suggestions/issues/8075
